zmon_aws_agent/aws.py

```diff
@@ -56,6 +57,7 @@ def base_encode(integer, base=BASE_LIST):
     return ret
 
 
+@trace(operation_name='populate_dns_data', tags={'aws', 'dns'})
```

Comment:
``tags={'aws': 'dns'}`` dict not a set

Reply:
operation_name is not needed since it is set by default in the ``@trace`` decorator.